### DISCUSSION TITLE
Creating Custom fieldsets as a non-administrator

### DISCUSSION THREAD

**Discussion by baknight1975** [cerb/cerb-release#32] [Q&A]
We have a user group that needs to be able to create "custom fieldsets".   We are using cloud version 10.0.1 and have configured a "role" with the create permission.  When we have the end-user try to create a new custom fieldset then get a "Error:Forbidden" message.  

![image](https://user-images.githubusercontent.com/62525891/120823386-61c9a980-c525-11eb-9450-10f6a0a98277.png)

  We are currently working around this by creating a temp one, assigning it to their role, then having them rename and setup it up as needed.   Do staff have to be Administrators to create fieldsets?  

Below is a snapshot of how their permissions look.

![image](https://user-images.githubusercontent.com/62525891/120823588-92a9de80-c525-11eb-82db-727fd4accbbc.png)

Thanks as always,
Barry

**Comment by jstanden** [comment]
Hi Barry!

I see a logic error in the code that's unnecessarily preventing this. It's checking if the worker is able to write the record when it's defaulted to being owned by 'Cerb' (which only an admin could do).

Since you're on Cerb Cloud, I can patch this and deploy it for you now.

Thanks for the report. 👍

**Comment by baknight1975** [comment]
Thanks so much Jeff.   That’s awesome.

You guys rock.

Barry

**Reply by jstanden** [reply]
Sure thing! I just deployed that patch. It should start working for you in a couple mins.
